raw-data/EA/EA_inventories.tsv

```diff
@@ -706,45 +706,45 @@ InventoryID	LanguageCode	LanguageName	Segment
 2255	dlg	Dolgan	ɛː
 2255	dlg	Dolgan	ɔː
 2255	dlg	Dolgan	ɑː
-2256	drh	Darkhat	b
```

Comment:
Note for posterity: this language code was retired from ISO639-3 on the grounds that it is more properly considered a dialect of Halh Mongolian (`khk`)